src/routes/docs/products/messaging/messages/+page.markdoc

```diff
@@ -68,8 +68,8 @@ This helps you reach your user, even when their device do not have internet acce
 {% /table %}
 
 # Messages lifecycle {% #messages-lifecycle %}
-All messages begin as `draft`.
-If the message is sent immediately, its status is set to `processing`.
+All messages begin as `draft`. 
```

Comment:
Ooops...technically, messages can go straight to processing. Sorry!

Reply:
Done